pkg/watch/health.go

```diff
@@ -57,8 +58,9 @@ type PodWatch struct {
 // runs a CheckHealth routine to monitor the health of each
 // service and kills routines for services that should no
 // longer be running.
-func MonitorPodHealth(config *preparer.PreparerConfig, logger *logging.Logger, shutdownCh chan struct{}) {
+func MonitorPodHealth(config *preparer.PreparerConfig, shutdownCh chan struct{}) {
```

Comment:
I originally removed it to help figure out where the nil pointer dereference error was coming from. It had nothing to do with the error, but I left the change in because it simplifies the MonitorPodHealth API. Though now that I think about it thats not really a good thing cause it makes the logging opaque and inflexible for the client.

I will revert this.